### PR TITLE
add basket ui

### PR DESCRIPTION
**Pull request by gomelviiv**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Awesome App</title>
+</head>
+<body>
+  <div id="app"></div>
+  <script src="dist/bundle.js"></script>
+</body>
+</html>
```
Comment (angelinabylitskaya) [inline index.html:13]: image

File: src/components/BasketItem/BasketItem.tsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+function BasketItem() {
+  return (
+    <div className="cart__item">
+      <div className="cart__item-img">
+        <img src="../assets/img/ring.png" alt="" />
+      </div>
+      <div className="cart__item-info">
+        <h3>Name Eman</h3>
+        <p>Категория: Подвеска, Бренд: Name Eman</p>
```
Comment (angelinabylitskaya) [inline src/components/BasketItem/BasketItem.tsx:15]: 2 languages are used - bad.
Use English or add multi language text.
https://medium.com/@ricklee_10931/react-multi-lingual-with-react-i18next-57879f986168

File: src/components/Header/HeaderMenu/HeaderMenu.tsx [hunk 3]
```diff
@@ -92,7 +93,9 @@ function HeaderMenu() {
             <a href="#">Вакансии</a>
           </div>
           <div>
-            <img src="./assets/img/OJJo.png" alt="" />
+            <Link to="/">
+              <img src="./assets/img/OJJo.png" alt="" />
```
Comment (angelinabylitskaya) [inline src/components/Header/HeaderMenu/HeaderMenu.tsx]: Add alt text.

File: src/styles/components/_banner.scss [hunk 4]
```diff
@@ -0,0 +1,10 @@
+.banner {
+  background-color: $secondary-color;
+  box-sizing: border-box;
+  padding: 2rem;
+  text-align: center;
+
+  &__text {
+    color: $main-color;
+  }
+}
```
Comment (angelinabylitskaya) [inline src/styles/components/_banner.scss]: No newline

Comment (angelinabylitskaya) [reply src/styles/components/_banner.scss]: Add prettier to avoid code style issues.

File: src/styles/components/_basket_item.scss [hunk 5]
```diff
@@ -0,0 +1,94 @@
+.cart__item {
+  @include typeFlex(row, center, space-between);
+  margin-top: 50px;
+  .cart__item-img {
```
Comment (angelinabylitskaya) [inline src/styles/components/_basket_item.scss]: For more readability add empty line before each selector.

File: src/styles/reset.scss [hunk 6]
```diff
@@ -0,0 +1,78 @@
+/* Указываем box sizing */
```
Comment (angelinabylitskaya) [inline src/styles/reset.scss:1]: User comments only on English.

File: src/styles/sections/_bottom_basket.scss [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+  border-top: 1px #e6e4e4 solid;
+  margin: 50px 0;
+  .basket__bottom-details {
+    @include typeFlex(row, center, space-between);
+    & span {
+      font-size: 22px;
+    }
+    & b {
+      margin-left: 20px;
+      color: #232323;
```
Comment (angelinabylitskaya) [inline src/styles/sections/_bottom_basket.scss]: Move Colors to a variable file so they can be easily changed.

File: src/styles/variables.scss [hunk 8]
```diff
@@ -0,0 +1,5 @@
+$main-font: Gilroy;
+$secondary-font: Gilroy Serif;
+
+$main-color: #fff;
+$secondary-color: #333333;
```
Comment (angelinabylitskaya) [inline src/styles/variables.scss:5]: No newline

File: src/styles/sections/_bottom_basket.scss [hunk 9]
```diff
@@ -0,0 +1,50 @@
+.basket__bottom {
+  padding-top: 20px;
+  border-top: 1px #e6e4e4 solid;
+  margin: 50px 0;
+  .basket__bottom-details {
```
Comment (angelinabylitskaya) [inline src/styles/sections/_bottom_basket.scss]: You can use parent ref here.
`&-details {`

File: src/styles/sections/_bottom_basket.scss [hunk 10]
```diff
@@ -0,0 +1,50 @@
...
+    & span {
+      font-size: 22px;
+    }
+    & b {
+      margin-left: 20px;
+      color: #232323;
+      font-weight: 600;
+    }
+  }
+  .basket__bottom-buttons {
```
Comment (angelinabylitskaya) [inline src/styles/sections/_bottom_basket.scss]: `&-buttons {`

File: src/styles/sections/_bottom_basket.scss [hunk 11]
```diff
@@ -0,0 +1,50 @@
...
+    @include typeFlex(row, center, space-between);
+    margin-top: 40px;
+    .basket-back-btn {
```
Comment (angelinabylitskaya) [inline src/styles/sections/_bottom_basket.scss:24]: Here also can be used the following code. Avoid too deep nesting.
```
.basket {

  &-back-btn {...}

  &-buy {...}
}
```

**Review by angelinabylitskaya — CHANGES_REQUESTED** [review]

**Comment by gomelviiv** [comment]
@angelinabylitskaya, I tried to fix all issues. In my next feature I correct all project by your template.

**Pull request merged**